lib/top_hangman/round.rb

```diff
@@ -1,9 +1,23 @@
+require_relative 'render'
+
 module TopHangman
   class Round
     attr_reader :word
 
     def initialize(word)
       @word = word
+      @end_of_round = false
+      @guesses = []
+      @guess = nil
+      @guess_count = 0
+      @word_array = word.split('')
+      @word_progress = Array.new(@word_array.length, '_')
```

Comment:
I think it would be cool to have another abstraction, called `TargetWord` that has an attribute called `word`, a method called `array` or `to_array`, and another attribute called  `progress`. What do you think?

Reply:
Or even better, I see you have some more logic on the Game class around words. What if you create a `Word` class instead of `TargetWord`, and use it both here and there?